Gopkg.toml

```diff
@@ -1,6 +1,6 @@
 [[constraint]]
   name = "github.com/DataDog/agent-payload"
-  version = "4.13.0"
+  revision = "cb184060464f95df4a0610585953eec30adf186d"
```

Comment:
This needs to be updated to once https://github.com/DataDog/agent-payload/pull/26 is merged